controllers/get-all-articles.controller.js

```diff
@@ -0,0 +1,9 @@
+const { selectAllArticles } = require("../models/topics.models")
+
+const getAllArticles = (req, res) => {
+    selectAllArticles().then((articles) => {
+    res.status(200).send(articles)
```

Comment:
You should be sending your articles back on an appropriate key for example `{articles}`